### PR TITLE
The installation guide should be easier to get to

### LINKED ISSUE #1730: The installation guide should be easier to get to
Currently, a link to the [installation instructions](https://elementary.io/docs/installation) is shown on the popup before the user downloads elementaryOS. However, users might be inclined 
to just click the big, colorful button without reading the text above it. Furthermore, users might want to download first, while they have the dialog open, and follow the instructions later. After users click the button, they are abruptly brought back to the home page. Then, they might be confused because they can't open the disk image with their file manager. Worse, they might find unsafe or difficult instructions online. In any case, it should be more obvious that there is an easy installation guide available.

Possible solutions:
- Make the "installation guide" link more obvious (make it bigger, more colorful, etc)
- Automatically redirect the user to the installation guide after the "download" button is clicked (the user probably doesn't need the homepage after installing anyway)
- Show a page after downloading that says something like "Thank you for downloading! To install, follow [these instructions]()"

### PR DESCRIPTION
**Pull request by brunocamarggo**
Fixes #1730 

### Changes Summary

-  Show a page after downloading that says something like "Thank you for downloading! To install, follow these instructions" [**done**]

This pull request is ready for review.

### CODE REVIEW

**Comment by brunocamarggo** [comment]
This is a new pull request because I delete my forked repository by mistake. Sorry guys.

**Comment by jwinnie** [comment]
@elementary Is someone working on the design? If not, how could I participate?

**Comment by lewisgoddard** [comment]
It requires approval from the design team, two of which have been tagged for review.

File: thank-you.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    $page['title'] = 'Thank You for downloading';
+    $page['theme-color'] = '#3E4E54';   
+    
+    include $template['header'];
+    include $template['alert'];
+?>
+
+<section class="hero">
+    <div class="grid">
+    <div>
```
Comment (btkostner) [inline thank-you.php]: You seem to have double `divs` here

File: thank-you.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <img src="<?=$sitewide['root'];?>images/brand/logomark.png" alt="logomark">
```
Comment (btkostner) [inline thank-you.php]: We tend to not use PHP shorthand. Please change this to `<?php echo $sitewide['lang-root']; ?>`

File: thank-you.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <h1>Thank you for downloading!</sup></h1>
+        <h4>For help and more info please see the installation guide</h4>
+        <a class="button suggested-action" href="/docs/installation#installation">Read the installation guide</a>
```
Comment (btkostner) [inline thank-you.php]: You will want to add the `lang-root` to this link as well.

File: thank-you.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    include $template['header'];
+    include $template['alert'];
+?>
+
+<section class="hero">
+    <div class="grid">
+    <div>
+        <img src="<?=$sitewide['root'];?>images/brand/logomark.png" alt="logomark">
+        <h1>Thank you for downloading!</sup></h1>
+        <h4>For help and more info please see the installation guide</h4>
```
Comment (sumnerevans) [inline thank-you.php]: Grammatical suggestion:
> For help and more info, please see the installation guide

Note the added comma. I'm no grammar genius, but it *feels* more natural to me.

**Comment by cassidyjames** [comment]
@btkostner is it possible to get a test branch of this up on beta.elementary.io?

**Comment by jwinnie** [comment]
I think saying “installation guide” twice is redundant.

**Pull request merged**